### ISSUE TITLE
Breaking changes between v1.12.0 and v1.13.0!

### ISSUE THREAD

**Issue by Snawe** [crowdin/crowdin-api-client-python#165]
Hey guys!

After an update of the library from v1.7 to v1.15 some parts of my script broke. After quite some time checking the patch notes and some other parts of my script, I found out that you broke the interface of some functions by changing the order of the function arguments! IMHO this is a big nono, especially in minor version updates.

For better search (and maybe it helps someone else), the functions I had problems with are: `update_file` and `get_file_revision`

v1.12.0 
https://github.com/crowdin/crowdin-api-client-python/blob/1.12.0/crowdin_api/api_resources/source_files/resource.py#L345
https://github.com/crowdin/crowdin-api-client-python/blob/1.13.0/crowdin_api/api_resources/source_files/resource.py#L547

v1.13.0
https://github.com/crowdin/crowdin-api-client-python/blob/1.12.0/crowdin_api/api_resources/source_files/resource.py#L345
https://github.com/crowdin/crowdin-api-client-python/blob/1.13.0/crowdin_api/api_resources/source_files/resource.py#L547

Maybe there are more... but for me it failed on those two.

In the future, please **don't** change the order of arguments. If, for what ever reason, it is required, mark it as breaking version.

**Comment by andrii-bodnar** [comment]
Hi @Snawe!

I just checked the release changelog and I think #129 is the culprit in this issue. This PR was included in the [1.13.0](https://github.com/crowdin/crowdin-api-client-python/releases/tag/1.13.0) release. It actually brings a breaking change in some methods that have the project ID in the argument list.

I just added a warning to the release notes for version 1.13.0.

We'll pay more attention to this kind of thing in the future. Thanks for pointing this out and sorry for the inconvenience! 🙌
